packages/venia-concept/src/ServiceWorker/setupWorkbox.js

```diff
@@ -1,9 +1,12 @@
+import { skipWaiting, clientsClaim } from 'workbox-core';
+import { precacheAndRoute } from 'workbox-precaching';
+
 export default function() {
     /**
      * Import and Instantiate workbox object.
      */
     importScripts(
-        'https://storage.googleapis.com/workbox-cdn/releases/4.3.1/workbox-sw.js'
+        'https://storage.googleapis.com/workbox-cdn/releases/5.1.2/workbox-sw.js'
```

Comment:
Does this version (`5.1.2`) differ from the version listed in `package.json` (`5.1.3`) for any particular reason?

Reply:
Oh that was actually my bad - I need to update this. Thanks for reminding me :)